vnext/Microsoft.ReactNative/Modules/Animated/InterpolationAnimatedNode.cpp

```diff
@@ -24,49 +24,66 @@ InterpolationAnimatedNode::InterpolationAnimatedNode(
   m_extrapolateRight = config.find(s_extrapolateRightName).dereference().second.asString();
 }
 
-void InterpolationAnimatedNode::Update() {}
+void InterpolationAnimatedNode::Update() {
+  assert(!m_useComposition);
+  if (m_parentTag == s_parentTagUnset) {
+    return;
+  }
+
+  if (const auto manager = m_manager.lock()) {
+    if (const auto node = manager->GetValueAnimatedNode(m_parentTag)) {
+      RawValue(Interpolate(node->Value()));
+    }
+  }
+}
 
 void InterpolationAnimatedNode::OnDetachedFromNode([[maybe_unused]] int64_t animatedNodeTag) {
   assert(m_parentTag == animatedNodeTag);
   m_parentTag = s_parentTagUnset;
-  m_propertySet.StopAnimation(s_valueName);
-  m_propertySet.StopAnimation(s_valueName);
-  m_rawValueAnimation = nullptr;
-  m_offsetAnimation = nullptr;
+
+  if (m_useComposition) {
+    m_propertySet.StopAnimation(s_valueName);
+    m_propertySet.StopAnimation(s_valueName);
```

Comment:
Unrelated bug: looks like the value animation is stopped twice, rather than stopping the animation on both the value and offset animations.